treeherder/perf/models.py

```diff
@@ -15,13 +15,17 @@
 class PerformanceFramework(models.Model):
 
     name = models.SlugField(max_length=255, unique=True)
+    enabled = models.BooleanField(default=False)
 
     class Meta:
         db_table = 'performance_framework'
 
     def __str__(self):
         return self.name
 
+    def __getitem__(self, enabled):
```

Comment:
It allows to access 'enabled' field in the framework. If it is not included the following error pops up while running the tests:

File "/home/vagrant/treeherder/treeherder/etl/perf.py", line 98, in _load_perf_artifact
    if framework['enabled'] is False:
TypeError: 'PerformanceFramework' object has no attribute '__getitem__'